packages/turf-difference/index.js

```diff
@@ -43,22 +43,16 @@ function difference(polygon1, polygon2) {
     var geom2 = getGeom(polygon2);
     var properties = polygon1.properties || {};
 
-    // Issue #721 - JSTS can't handle empty polygons
+    // Issue #721 - JSTS/Martinez can't handle empty polygons
```

Comment:
😢 Same issue is occuring with Martinez?...

Reply:
Yes the issue still occurs in Martinez but I didn't dig into it too deeply at 11pm so I'll look into it further